automation_tools/__init__.py

```diff
@@ -2296,6 +2299,8 @@ def product_install(distribution, create_vm=False, certificate_url=None,
         **installer_options
     )
 
+    execute(run_command, os.environ.get('FIX_POSTINSTALL'), host=host)
```

Comment:
any reason why putting this here and not after executing all below tasks?
e.g. what if i wanted to implement a temporary fix for foreman_discovery? its task is executed after this line and would roll-over my temp changes

Reply:
lets say we wanted to fix hammer for using server as a hostname / instead of localhost
then doing it after all postinstall task wouldn't have helped.

I'm open to have more injection points 
- prerpm 
- postrpm = preinstall
- postinstall
- pluginpostinstall < you are talking about this one

But didn't want to scare all out with so many options :)